README.md

```diff
@@ -1,12 +1,92 @@
 # Static Website Template
 Use this template to create a static website that is published over a CDN.
 
-The template is made of two resources: 
+## Getting started
+
+### Things you'll need 
+1. AWS [SAM-CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+1. An Altostra Account( Don't have one yet? Just [login](https://app.altostra.com) here)
+1. Altostra CLI Installed ([how?](../reference/CLI/altostra-cli.html#installation))
+1. VSCode with Altostra Editor ([how?](../getting-started/installation.html#install-the-visual-studio-code-extension))
+1. A Connected AWS Account ([how?](../getting-started/connect-your-accounts.html#connect-your-cloud-service-accounts))
+1. An Environment connected to your AWS Account ([how?](../howto/envs/manage-environments.html)) - We'll call it `Dev` for berevity, but you can pick any of your environments
+
+### Using the template
+
+You have several options to get started with this template.  
+Either go to the Altostra Web Console and create a new project.  
+When asked to use a template, select "Static Website".
+
+Alternatively, you can use the Altostra CLI to initialize a new project from the template by running:
+```sh
+$ alto init --template static-website
+```
+
+You can also apply the template to an existing Altostra project from Visual Studio Code by going 
+to the Altostra view in the main toolbar and clickign on "Static Website" in the templates list.
+
+### Required configuration
+
+This template contains a few parameters that must be 
+[set in the the environment](https://docs.altostra.com/howto/envs/manage-environments.html#manage-configuration-values)
+in which the deployment was created.
+
+You may also provide all or part of the configuration when 
+[creating or updating the deployment](https://docs.altostra.com/reference/CLI/commands/altostra-cli-deploy.html#providing-configuration).
+
+#### Template parameters
+
+- `DOMAIN`:  
+The fully qualified root domain of your application or arganization.
+- `DOMAIN_HOST_NAME`:  
+A host name for the site CDN.  
+It is combined with `DOMAIN` to create a fully qualified domain for the CDN.  
+I.E. If we set `DOMAIN: example.com` and `DOMAIN_HOST_NAME: my-site` the CDN would
+accept requests for `my-site.example.com`.
+- `DOMAIN_CERT`:  
+An Amazon certificate manager (ACM) certificate ARN approprite for the domain.
+
+### Project deployment
+
+Here are simple instruction to get you started.  
+For more information on each command please consult [Altostra docs](https://docs.altostra.com/reference/CLI/altostra-cli.html).
+
+1. Create an image from the project:
+```shell
+$ altos push v1.0
+```
+2. Deploy the image to a new deployment named `main` in the `Dev` environment:
+```shell
+$ alto deploy main:v1.0 --new Dev
+```
+3. Update the deployment with new images:
+```shell
+$ alto deploy main:v2.0
```

Comment:
No need for an update if nothing has changed. You make it look like it's part of the required steps.
A better step would be to run `alto deployments main`.